roles/teslamate/defaults/main.yml

```diff
@@ -0,0 +1,158 @@
+#########################################################################
+# Title:            Sandbox: teslamate                                  #
+# Author(s):        maindust                                            #
+# URL:              https://github.com/saltyorg/Sandbox                 #
+# --                                                                    #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+# defaults/main.yml
+
+teslamate_name: teslamate
+
+################################
+# Settings
+################################
+
+TM_DB_USER: "{{ user.name }}"
+TM_DB_PASS: "password4321"
```

Comment:
As above regarding var naming. Consider using Saltbox secrets to generate and store a random password